dependency.do

```diff
@@ -3,3 +3,4 @@ cap ssc install ftools
 cap ssc install ivreghdfe
 cap ssc install ivreg2
 cap ssc install ranktest
+cap ssc install avar
```

Comment:
@Constantino-Carreto-Romero If I am not mistaken, `ivreg2` uses `avar` for some standard error calculations. If that's the case, then the extra `avar` dependency may be redundant.

On a related note, can we add version numbers for the packages as comments in this file, please?

Reply:
I checked the documentation and ado files and the only `ivreg2`'s dependency is `ranktest`, but `ivreg2` does't install `ranktest` automatically, so it is necessary to keep `ranktest` in the dependency list. On the other hand, `ranktest` does require `avar`, but it doesn't install it automatically, so it is necessary to keep `avar` in the dependecy list. 

In https://github.com/JMSLab/xtevent/commit/c18b7d6db44fc5bc989a941255eca19f739c8d9d, I added the version numbers.